ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectSSOTest.kt

```diff
@@ -130,14 +135,49 @@ class RedirectSSOTest : StringSpec() {
             }
         }
 
+        "Redirect AuthnRequest Using DSA1 Signature Algorithm" {
+            Log.debugWithSupplier { "Redirect AuthnRequest Using DSA1 Signature Algorithm" }
+            CURRENT_SP_ISSUER = DSA_SP_ISSUER
+            CURRENT_SP_ENTITY_INFO = DSA_SP_ENTITY_INFO
+
+            val authnRequest = createDefaultAuthnRequest(SamlProtocol.Binding.HTTP_REDIRECT)
+
+            val encodedRequest = encodeAuthnRequest(authnRequest)
+            val queryParams = SimpleSign(DSA_SP_ISSUER).signUriString(
+                    SAML_REQUEST,
+                    encodedRequest,
+                    null)
+
+            // Get response from AuthnRequest
+            val response = sendRedirectAuthnRequest(queryParams)
+            BindingVerifier.verifyHttpStatusCode(response.statusCode)
+
+            val finalHttpResponse =
+                    getServiceProvider(IdpSSOResponder::class).getResponseForRedirectRequest(
+                            response)
+            val samlResponseDom = finalHttpResponse.getBindingVerifier().decodeAndVerify()
+
+            CoreAuthnRequestProtocolVerifier(authnRequest, samlResponseDom).apply {
+                verify()
+                verifyAssertionConsumerService(finalHttpResponse)
+            }
+            SingleSignOnProfileVerifier(authnRequest, samlResponseDom).apply {
+                verify()
+                verifyBinding(finalHttpResponse)
+            }
+
+            CURRENT_SP_ISSUER = DEFAULT_SP_ISSUER
```

Comment:
❓ Do we really need to set it back? If this test passes that mean the IdP supports DSA so we can use that for the remaining tests.